packages/core/src/messagingProtocols/proteus/ProteusService/ProteusService.test.ts

```diff
@@ -225,36 +237,24 @@ describe('ProteusService', () => {
     it('returns encrypted payload', async () => {
       const {
         services,
-        data: {firstUser, validPreKey, encryptedMessageBuffer, messageBuffer},
+        data: {firstUser, encryptedMessageBuffer, messageBuffer},
       } = await prepareDataForEncryption(false);
 
       const userClients: UserClients = {
         [firstUser.id]: [firstUser.clients.first, firstUser.clients.second],
       };
 
-      const preKeyBundleMap: UserPreKeyBundleMap = {
-        [firstUser.id]: {
-          [firstUser.clients.first]: validPreKey,
-          [firstUser.clients.second]: validPreKey,
-        },
-      };
-
       const encryptedPayload = new Map([
         [firstUser.sessions.first, encryptedMessageBuffer],
         [firstUser.sessions.second, encryptedMessageBuffer],
       ]);
 
-      jest
-        .spyOn(services.apiClient.api.user, 'postMultiPreKeyBundles')
-        .mockImplementationOnce(() => Promise.resolve(preKeyBundleMap));
-
```

Comment:
Doing the request to create the session from prekeys and getting the prekeys from backend are already covered in the SessionHandler tests